node/lib/util/write_repo_ast_util.js

```diff
@@ -61,8 +61,14 @@ const TestUtil            = require("../util/test_util");
  * @return {String}
  */
 const doExec = co.wrap(function *(string) {
-    const result = yield exec(string);
-    return result.stdout.split("\n")[0];
+    try {
+        const result = yield exec(string);
+        return result.stdout.split("\n")[0];
+    }
+    catch (e) {
+        console.log(string.split("\n")[1]);
```

Comment:
The variable named "string" here should  be called "command".  And maybe just print the whole thing?